packages/shared/src/components/comments/CommentInputOrModal.tsx

```diff
@@ -11,8 +11,9 @@ import { useMutateComment } from '../../hooks/post/useMutateComment';
 
 interface CommentInputOrModalProps
   extends Partial<LazyModalCommonProps>,
-    Omit<CommentMarkdownInputProps, 'className'> {
+    Omit<CommentMarkdownInputProps, 'className' | 'onRequestClose'> {
```

Comment:
Oh, it seems that we won't need the new property after all. Looks like we already have an `onClose` property which we can use.
```suggestion
    Omit<CommentMarkdownInputProps, 'className'> {
```